resources/views/layouts/app.blade.php

```diff
@@ -6,29 +6,35 @@
 
         <link rel="stylesheet" href="/css/app.css">
 
-        <link href="//fonts.googleapis.com/css?family=Lato:100" rel="stylesheet" type="text/css">
+        <link href="https://fonts.googleapis.com/css?family=Roboto:300,400" rel="stylesheet" type="text/css">
 
         <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">
 
     </head>
     <body>
-        <div class="container">
-            <div class="row">
-                @if (Auth::check())
-                <a href="/logout" class="btn btn-default pull-right meta-button">Log out <i class="fa fa-sign-out"></i></a>
-                @else
-                <a href="/auth/github" class="btn btn-default pull-right meta-button"><i class="fa fa-github"></i> Log in with GitHub</a>
-                @endif
-                <h1 class="text-center page-title page-title"><img src="/giscus-logo.png" class="giscus-logo giscus-logo" alt="Giscus"></h1>
-            </div>
-            <div class="row col-md-8 col-md-push-2">
-                @yield('content')
+        <header>
```

Comment:
I'd prefer we give this a class and make it a div, instead of styling it just based on being `header`